operators/advanced_network/advanced_network/managers/dpdk/adv_network_dpdk_mgr.h

```diff
@@ -198,6 +198,7 @@ class DpdkMgr : public Manager {
   uint64_t get_burst_tot_byte(BurstParams* burst) override;
   BurstParams* create_tx_burst_params() override;
   bool validate_config() const override;
+  uint16_t get_num_rx_queues(int port_id) const override;
```

Comment:
Why is the generic one not good?

Reply:
in `advanced_network::Manager`

Reply:
DPDK can create some hidden queues when they don't have either TX or RX defined. This gets the correct number once it's adjusted.

Reply:
Can you add that comment in the implementation?